### ISSUE TITLE
Disable auto close of expandable item

### ISSUE THREAD

**Issue by beczesz** [mikepenz/MaterialDrawer#2512]
Hi Mike, thanks for this excellent lib.

Is there a simple and elegant way to prevent to auto close expandable items when I refresh the elements in the list?

Thak you

**Comment by mikepenz** [comment]
how do you refresh the elements in the list? 

the best would be to get the expanded elements, and expand them again.
Note that there is a lot of trickery as expandable items might move if you add new elements :/

**Comment by mikepenz** [comment]
you could theoretically use:

`saveInstanceState` to save the identifiers of the expanded elements

 and `withSavedInstanceState` to reapply the same

**Comment by mikepenz** [comment]
Have you had a chance to try out this approach?

**Comment by beczesz** [comment]
Hi Mike, sorry for the late reply. I had to implement my own tree hierarchy and we decided internally that for now only the root view will be expandable. However with that I just left it as it is (as soon as the view refreshes it collapses). Most probably I will reiterate and fix that feature once I finish some more urgent tasks and then I will let here a comment with the results.

My suspicion is that it will be working as you suggested.
